.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,17 @@
+<!-- This form is for bug reports and feature requests ONLY! 
+
+If you're looking for help check [Vault's Google Group](https://groups.google.com/forum/#!forum/vault-tool).
+-->
+
+**Type of report:**
+
+[ ] Bug
+[ ] Feature request
+
+**Environment:**
```

Comment:
It's unclear what the difference is between environment and platform. Maybe combine into Operating System/Architecture?